parser.go

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cassandra
+
+import (
+	"fmt"
+	"strings"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+const (
+	insertQuery = "INSERT INTO %s (%s) VALUES (%s)"
+	updateQuery = "UPDATE %s SET %s WHERE %s"
+	deleteQuery = "DELETE FROM %s WHERE %s"
+
+	setStatementSeparator   = ","
+	whereStatementSeparator = "AND"
+)
+
+// Parser parses a record into a CQL query statement and its values
+type Parser struct{}
```

Comment:
I'm not sure `Parser` is the right name for this struct, think about renaming to something like `QueryBuilder`.

Reply:
I was not happy with the name either! `QueryBuilder ` sounds great